packages/formatter-html/src/result.ts

```diff
@@ -174,15 +171,12 @@ export class CategoryResult {
         let hint = this.getHintByName(hintId);
 
         if (!hint) {
-            // All the problems have to have the same severity, so we just need to calculate the status once.
-            const status = problem.severity === Severity.error ? 'error' : 'warning';
-
-            hint = new HintResult(hintId, status, this.url, this.isScanner);
+            hint = new HintResult(hintId, this.url, this.isScanner);
```

Comment:
How are we indicating the severity of each report? Can you paste a screenshot?

Reply:
We don't do that in this formatter